modular_skyrat/modules/cyborg/code/mechafabricator_designs.dm

```diff
@@ -15,3 +15,12 @@
 	materials = list(/datum/material/iron = 14500, /datum/material/glass = 7500, /datum/material/silver = 6000, /datum/material/gold = 1500,  /datum/material/diamond = 200, /datum/material/titanium = 8000, /datum/material/plasma = 2000)
 	construction_time = 80
 	category = list("Cyborg Upgrade Modules")
+
+/datum/design/affection_module
+	name = "Cyborg Upgrade (Affection Module)"
+	id = "affection_module"
+	build_type = MECHFAB
+	build_path = /obj/item/borg/upgrade/affectionmodule
+	materials = list(/datum/material/iron=4000, /datum/material/glass=1000)
```

Comment:
That's not exactly what I'd call a "cheap" upgrade, especially for early-game.

Reply:
I mean to be fair I dont really watch resources since I dont play roles that often stare at it (well that plus I play in engineering and put all the spare matts into the silo round start so there's usually a good ton)